cmd/perennial_branches.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Originate/git-town/lib/git"
+	"github.com/spf13/cobra"
+)
+
+var branchToAdd string
+var branchToRemove string
+
+var perennialBranchesCommand = &cobra.Command{
+	Use:   "perennial-branches",
+	Short: "Displays or updates your perennial branches",
+	Run: func(cmd *cobra.Command, args []string) {
+		if branchToAdd != "" {
+			git.EnsureHasBranch(branchToAdd)
```

Comment:
Extract this block into an `addPerennialBranch(branchName)` method in this file.